requirements-freeze.txt

```diff
@@ -8,10 +8,10 @@ certifi==2021.5.30
 cffi==1.14.6
 chardet==3.0.4
 -e git+https://github.com/ckan/ckan.git@e50b79651090c5d5946660a4107915c5d59dda86#egg=ckan
--e git+https://github.com/GSA/inventory-app.git@9eeebdb5a7ee41835dbc5ef8f2807beb40c28fce#egg=ckanext_datagov_inventory&subdirectory=ckanext-datagov_inventory
+-e git+https://github.com/GSA/inventory-app.git@38a345c679037ecf63ed09917d1fa685e429867f#egg=ckanext_datagov_inventory&subdirectory=ckanext-datagov_inventory
 -e git+https://github.com/GSA/ckanext-datajson.git@4ca1ca31cfcc23d6610c13693049fcfc83696d08#egg=ckanext_datajson
-ckanext-dcat-usmetadata==0.2.21
--e git+https://github.com/GSA/ckanext-googleanalyticsbasic@f0b75cf965e477a84f044885b27880782a48969c#egg=ckanext_googleanalyticsbasic
+ckanext-dcat-usmetadata==0.3.0
```

Comment:
A decent amount of things changed in dcat_usmetadata in 0.3.0... I'm surprised that we didn't merge this update before.  Just keep an eye on it.

Reply:
👀